Expensify/App
# [No QA] Display workspaces in settings page

### Details
Displays free policies (workspaces) in the settings page.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/166038

### Tests
1. Make sure you pull and rebuild auth.
1. Comment out [these lines](https://github.com/Expensify/Web-Expensify/blob/51cfdd579b022b651f732860568f0b8b0b8ac506/api.php#L89-L91) on dev.
1. Go to https://expensify.com.dev/ and sign in
1. Go to https://www.expensify.com.dev/api?command=Policy_Create&policyName=WorkspaceTest&type=free to create a new policy w/ type `free`
1. Run E.cash and sign into the same account you used on E.com
1. Go to settings, verify that you see workspaces at the top of the list.
1. Repeat step 4 (except with a different policy name)
1. Verify that you see both free policies.

### QA Steps
No QA for now.

### Tested On


### Screenshots

#### Web
![image](https://user-images.githubusercontent.com/47436092/122311373-e376eb00-cec6-11eb-9717-f04fabc83d57.png)
![image](https://user-images.githubusercontent.com/47436092/122326849-cd2a5880-cee1-11eb-92a2-86a214bb23f8.png)

#### Mobile Web
![image](https://user-images.githubusercontent.com/47436092/122327028-124e8a80-cee2-11eb-83d8-41145326acdc.png)

#### Desktop
![image](https://user-images.githubusercontent.com/47436092/122327209-59d51680-cee2-11eb-88fb-860390fe9a32.png)

#### iOS
![image](https://user-images.githubusercontent.com/47436092/122327731-35c60500-cee3-11eb-9f4b-9223386c2077.png)

#### Android
